sunpy/net/dataretriever/tests/test_noaa.py

```diff
@@ -29,17 +73,18 @@ def test_get_url_for_time_range(timerange, url_start, url_end):
 def test_can_handle_query():
     ans1 = noaa.NOAAIndicesClient._can_handle_query(
         Time('2012/8/9', '2012/8/10'), Instrument('noaa-indices'))
-    assert ans1 == True
+    assert ans1 is True
```

Comment:
rather than using `is` here it would be better to just assert the variable.
```suggestion
    assert ans1
```